config/prow/config.yaml

```diff
@@ -586,6 +586,9 @@ branch-protection:
               - Test Linux
               - Test MacOS
               - Test Windows
+        lwkd:
+          exclude:
+            - "^20\d+" # don't protect weekly article dev branch
```

Comment:
```suggestion
            - "^20\\d+" # don't protect weekly article dev branch
```

Reply:
@jberkus, the backward slash also need escaping in the expression.

Reply:
Ah, thanks.